src/App.js

```diff
@@ -41,7 +41,18 @@ function App({ data }) {
     setActivities(newActivities);
     saveToLocal("localActivities", newActivities);
   }
+  //------------Delete Button -----------------
```

Comment:
I think you should delete this comment. :)

Reply:
Thx, I totally forgot that 😂